src/elements/hx-tablist/index.spec.js

```diff
@@ -0,0 +1,127 @@
+            const mockup = `
+                <hx-tabset current-tab="0">
+                    <hx-tablist>
+                        <hx-tab id="tab-1" current="true"></hx-tab>
+                        <hx-tab id="tab-2"></hx-tab>
+                        <hx-tab id="tab-3"></hx-tab>
+                    </hx-tablist>
+                    <hx-tabcontent>
+                        <hx-tabpanel id="panel-1" open></hx-tabpanel>
+                        <hx-tabpanel id="panel-2"></hx-tabpanel>
+                        <hx-tabpanel id="panel-3"></hx-tabpanel>
+                    </hx-tabcontent>
+                </hx-tabset>`;
+
+            const elSelector = 'hx-tabset';
+            const fragment = /** @type {HXTablistElement} */ await fixture(mockup);
+            const queryId = fragment.querySelector(elSelector).id;
+
+            expect(queryId).to.not.be.null;
+        });
+
+        it('should FAIL on render with NO initial <hx-tab>s', async () => {
+            const mockup =`
+                <div class="hxPanel hxTabbed">
+                    <hx-tabset id="tab-component-tests">
+                        <hx-tablist id="tablist">
+                        </hx-tablist>
+                        <hx-tabcontent id="tabcontent">
+                          <hx-tabpanel></hx-tabpanel>
+                        </hx-tabcontent>
+                    </hx-tabset>
+                </div>`;
+
+            const fragment = /** @type {HXTablistElement} */ await fixture(mockup);
+            const currentTabId = fragment.querySelector('hx-tab').id;
+
+            expect(currentTabId).to.be.null;
+        });
+
+        it('should FAIL on render with NO initial <hx-tabpanel>s', async () => {
+            const mockup =`
+            <div class="hxPanel hxTabbed">
+                <hx-tabset id="tab-component-tests">
+                    <hx-tablist id="tablist">
+                        <hx-tab id="tab-1" current="true"></hx-tab>
+                    </hx-tablist>
+                    <hx-tabcontent id="tabcontent">
+                    </hx-tabcontent>
+                </hx-tabset>
+            </div>`;
+
+            const fragment = /** @type {HXTablistElement} */ await fixture(mockup);
+            const queryId = fragment.querySelector('hx-tabpanel').id;
+
+            expect(queryId).to.be.null;
+        });
+
+        it('should FAIL on render with NO tabs or tabpanels', async () => {
+            const mockup = `
+                <div class="hxPanel hxTabbed">
+                    <hx-tabset id="tabTest">
+                        <hx-tablist>
+                        </hx-tablist>
+                        <hx-tabcontent>
+                        </hx-tabcontent>
+                    </hx-tabset>
+                </div>`;
+
+            const fragment = /** @type {HXTablistElement} */ await fixture(mockup);
+            const queryTabs = fragment.tabs.length;
+
+            expect(queryTabs).to.equal(0);
+        });
+    });
```

Comment:
#### @BY00565233, please add this block of tests customized to the your component tests - PR #732 and PR #709 (replace the `@type`).